.github/workflows/build.yml

```diff
@@ -65,6 +65,17 @@ jobs:
           path: playwright-report/
           retention-days: 30
 
+      # SonarCloud analysis
+      - name: SonarCloud
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
```

Comment:
Unnecessary to re-do the checkout -- move the fetch-depth:0 to the first checkout